Cod3rsGrowth.Infra/Repositorio/TesteDeJogoRepositorio.cs

```diff
@@ -1,11 +1,19 @@
 ﻿using Cod3rsGrowth.Dominio.Entidades;
 using Cod3rsGrowth.Infra.Interfaces;
+using LinqToDB;
 using System.Collections.Generic;
+using System.Linq;
 
 namespace Cod3rsGrowth.Infra.Repositorio
 {
     public class TesteDeJogoRepositorio : ITesteDeJogoRepositorio
     {
+        DbCod3rsGrowth bancoDeDados;
+
+        public TesteDeJogoRepositorio(DbCod3rsGrowth db)
+        {
+            this.bancoDeDados = db;
```

Comment:
mesma situação, n precisa dessa cláusula this se a variavel estiver privada no padrão do C#